agent/commands/clients.go

```diff
@@ -197,6 +202,8 @@ func serverRegister(cfgSetup *config.Setup) (agentID, token string, _ error) { /
 			Reregister:        cfgSetup.Force,
 			MetricsMode:       pointer.ToString(strings.ToUpper(cfgSetup.MetricsMode)),
 			DisableCollectors: disableCollectors,
+
+			ExistedServiceToken: existedServiceToken,
```

Comment:
not sure if we need to send this in the body of the request. We send it in headers, so we can read it from headers.

Reply:
Great idea. Removed. Now it is obtained from headers from ctx.